### PR TITLE
Tell people a little about how to get started using codesee

### PR DESCRIPTION
**Pull request by jleven**
Added
- How to get started
- Some pointers and caveats
- Compatibility: chrome only, third party cookies

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,16 +2,27 @@
...
-    * the CodeSee Tracker library `@codesee/tracker`, which actually sends the data to the CodeSee server
+In the instructions below, we will:
+
+1. Install two npm packages, and
+2. Configure babel so that CodeSee instruments your code when running in development
+
+Once installed, whenever you run your app in development and view it in your browser, you should see a circle with the CodeSee logo.
+
+- If it's red, click it to log into CodeSee
+- If it's purple, you're ready to go! Click it to start recording, interact with your app, then click it again to stop recording.
```
Comment (shaneakr) [inline docs/index.md:18]: Yep smart to add!

File: docs/index.md [hunk 2]
```diff
@@ -2,16 +2,27 @@
...
+
+A few thoughts to help you on your CodeSee journey:
+
+- Keep recordings as short as possible -- focus on the one thing you want to understand better.
+- When you stop recording, CodeSee will try to open a new tab with your recording in it shortly. For larger recordings or on slower networks, there can be some buffering, so please give it a few seconds or more to finish sending the recording to our servers.
+- The CodeSee UI can be a little sluggish with larger recordings! Sorry about that! For the moment, we ask that you give our UI a second to process now and then. But fear not, we take performance very seriously and we're actively working on performance improvements.
```
Comment (shaneakr) [inline docs/index.md:24]: should we add a note about popup blockers?

File: docs/index.md [hunk 3]
```diff
@@ -31,7 +42,7 @@ yarn add --dev @codesee/tracker@latest @codesee/babel-plugin-instrument@latest
 </details>
 
 ## CodeSee configuration for specific projects/environments
-We have specific instructions for certain projects and environment. If you are not using any of the following projects/environments, skip to the generic set up instructions:
+We have specific instructions for certain projects and environments. If you are not using any of the following projects/environments, skip to the Generic Setup Instructions near the end of this doc:
 
 - Create React App
 - Typescript, but without Babel
```
Comment (shaneakr) [inline docs/index.md:48]: should this actually be at the top 🤔

File: docs/index.md (hunk 2 above)
Comment (jleven) [reply docs/index.md]: Fixed!

**Review by shaneakr — APPROVED** [review]
approved!

**Pull request merged**